tests/features/tags/add-tag.spec.ts

```diff
@@ -40,89 +40,71 @@ test.describe('Tags', () => {
     });
     await test.step(`I mark checkbox 'important'`, async () => {
       await pages.modals.addTag.checkImportant(0);
-      expect(pages.modals.addTag.isCheckboxMarked(0)).toBeTruthy();
     });
     await test.step(`I click 'create' button`, async () => {
       await pages.modals.addTag.addNewTag.click();
       await pages.tags.waitUntilTagVisible(tagName);
       expect(await pages.tags.isTagVisible(`${tagName}`)).toBeTruthy();
-      expect(await pages.tags.isTagImportant(`${tagName}`)).toBeTruthy();
+      await expect(pages.tags.importantTag(tagName)).toBeVisible();
```

Comment:
Are you sure you need 'await' here? The 'importantTag' returns just a Locator, not a Promise.
Also, I recommend using the method .isVisible from CustomElement and changing the assertion on .toBeTruthy

Do it for all places in the MR

Reply:
Yep, you’re right about that two points above. 
My concern is only that the async function is slower than synchronous, which is why I recommend avoiding using it if have another way.

Reply:
> My concern is only that the async function is slower than synchronous

I think I got your point.
I think that using built-in async matchers will not increase execution time.

WHY?

Imagine, there are multiple assertions in the test. One by one:

```
await expect(locator1).toBeVisible();
await expect(locator2).toBeVisible();
await expect(locator3).toBeVisible();
```

In this case, the test will asynchronously wait until the first expectation will be completed (if the `locator1` was not properly loaded to the page).
But it is very likely that other elements (`locator2` and `locator3`) will already be available on the page. And the test will NOT wait a long time as in the case with `locator1`.

Reply:
For this case you can use Promise.all([]) and check all locators.